libraries/AP_HAL/examples/RCOutput/RCOutput.cpp

```diff
@@ -6,9 +6,13 @@
 
 const AP_HAL::HAL& hal = AP_HAL::get_HAL();
 
+#define RC_SPEED 50
+
 void setup (void) 
 {
     hal.console->println("Starting AP_HAL::RCOutput test");
+    hal.rcout->set_freq(0xFF, RC_SPEED);   // Set RC Speed
```

Comment:
Thanks for the feedback and suggestion - I'll have a look into it.  The only problem I see with a menu-type implementation would be the increase in inputs required by the user (start program > set rc_speed > run motors) as opposed to the very simple "just start the program and it goes" system at the moment.

The other option could be to change the #define RC_SPEED to 490 (instead of 50) to keep the functionality as it is now (not sure if/how that affects you guys though).